recipes/pymunk/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pymunk" %}
+{% set version = "5.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: d85ae4dc1bc6ee5a0a9ff6f5ff95b4f8a5892cf11985a2433a9eb52daf6515bf
+  
+build:
+  number: 0
+  # Note: --no-deps is currently required due to https://github.com/conda/conda-build/issues/3254
+  # Once resolved, it should be removed.
+  script: 
+    - export LDFLAGS="-dynamiclib $LDFLAGS"  # [osx]
+    - export LD=clang  # [osx]
+    - export CFLAGS="$CFLAGS -O3"  # [linux]
+    - export LDFLAGS="-shared $LDFLAGS"  # [linux]
+    - export LD=gcc  # [linux]
```

Comment:
These lines are ok for toolchain builds but will break the migration to the anaconda compilers.